extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -90,6 +90,9 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
         singlepart: dicomWebConfig.singlepart,
         headers: userAuthenticationService.getAuthorizationHeader(),
         errorInterceptor: errorHandler.getHTTPErrorHandler(),
+        largeStudySeriesCountThreshold:
```

Comment:
You aren't actually using the large study series count threshold anywhere, and that value doesn't belong in the data source, as it should simply fetch the initialSeriesFetchSize instances regardless of the study size.

Reply:
Removed both variables